lib/Smokeping.pm

```diff
@@ -3950,7 +3950,7 @@ Its location must be hardcoded in the smokeping script and smokeping.cgi.
 The contents of this manual is generated directly from the configuration
 file parser.
 
-The Parser for the Configuration file is written using David Schweikers
+The Parser for the Configuration file is written using David Schweikert
```

Comment:
This matches the other instance of his name in this project and the project per CPAN